src/stable/lib/ic_apis/call.ts

```diff
@@ -214,6 +232,15 @@ function handleTwoWay<Return>(
         );
         createRejectCallback(globalRejectId, reject);
 
```

Comment:
Why don't we need to check if azleIcStable is undefined anymore?

Reply:
But not strong enough that I want to do anything more than just say that here...